### PR TITLE
PHPStan (Level 5) on install-dev/

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | PHPStan (Level 5) on install-dev/
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Relative to #16471
| How to test?      | Github Actions are :green_circle: 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26819)

### CODE REVIEW

**Review by sowbiba — APPROVED** [review]
Perfect

**Review by matks — APPROVED** [review]

File: install-dev/controllers/http/configure.php [hunk 1]
```diff
@@ -150,14 +160,17 @@ public function processUploadLogo()
...
+                    // @phpstan-ignore-next-line
                     $white = imagecolorallocate($dest_image, 255, 255, 255);
+                    // @phpstan-ignore-next-line
                     imagefilledrectangle($dest_image, 0, 0, $src_width, $src_height, $white);
+                    // @phpstan-ignore-next-line
                     imagecopyresampled($dest_image, $src_image, 0, 0, 0, 0, $src_width, $src_height, $src_width, $src_height);
+                    // @phpstan-ignore-next-line
                     if (!imagejpeg($dest_image, _PS_ROOT_DIR_ . '/img/logo.jpg', 95)) {
-                        $error = $this->trans('An error occurred during logo copy.', [], 'Install');
+                        $error = $this->translator->trans('An error occurred during logo copy.', [], 'Install');
```
Comment (matks) [inline install-dev/controllers/http/configure.php:171]: What even is this code ? 😆  I dont find where we can upload a logo in the install process.

It's probably dead code ping @kpodemski @Quetzacoalt91 any memory of the install process supposed to allow logo upload?

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

File: install-dev/controllers/http/configure.php (hunk 1 above)
Comment (Quetzacoalt91) [reply install-dev/controllers/http/configure.php:171]: Yeah I remember such a feature in very old versions of PrestaShop! It was maybe PS 1.4

Comment (Progi1984) [reply install-dev/controllers/http/configure.php:171]: @sarahdib has a lot of archives 

image

Comment (Quetzacoalt91) [reply install-dev/controllers/http/configure.php:171]: THE ARCHIVIST! :file_cabinet:

**Pull request merged**